frontend/src/metabase/search/components/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -113,7 +113,9 @@ export const DropdownSidebarFilter = ({
           ) : (
             <Group noWrap>
               {iconName && <Icon name={iconName} />}
-              <Text weight={700}>{label}</Text>
+              <Text weight={700}>
```

Comment:
Adding this for the dynamic toggle label i.e. `Search SQL queries` vs `Search native queries`